### PR TITLE
New v.3 of Kay et al notebook, +updated version of CESM LENS landing page.

### PR DESCRIPTION
**Pull request by jeffdlb**

### CODE REVIEW

**Comment by jhamman** [comment]
@jeffdlb - can you summarize the changes in the notebook here?

**Comment by jeffdlb** [comment]
Summary of changes in v.3 notebook relative to previous:
* Additional comments and markdown to be a little more pedagogical.
* Defined at the top two constants (URLs of Intake Catalog and observational data file) that were previously hard-coded deep in the notebook.
* Inclusions of Brian Bonnlander's changes that fix the colorbar for Figure 4 and clean up the code there.

I will execute the notebook now that the binder hub is up again to verify it works properly.

**Comment by jeffdlb** [comment]
I'm getting an error in the Figure 4 creation at this line:
plot_data = winter_trends.TREFHT.isel(member_id = ensemble_index)

AttributeError: 'DataArray' object has no attribute 'TREFHT'

Will try running Brian's version, from which this code was taken and which previously appeared to work. Closing pull request until this is figured out.

**Comment by jeffdlb** [comment]
OK, I'm getting the same error in the currently-published Notebook in the master branch.
